.tx/config

```diff
@@ -3,8 +3,8 @@ host     = https://www.transifex.com
 lang_map = fa_AF: prs, uz@Cyrl: uz_UZ_Cyrl, uz@Latn: uz_UZ_Latn
 
 [o:hisp-uio:p:app-server-side-resources:r:dataentry-i18n-module-properties]
-file_filter  = dhis-2/dhis-web/dhis-web-dataentry/src/main/resources/org/hisp/dhis/de/i18n_module_<lang>.properties
-source_file  = dhis-2/dhis-web/dhis-web-dataentry/src/main/resources/org/hisp/dhis/de/i18n_module.properties
+file_filter  = dhis-2/dhis-web-dataentry/src/main/resources/org/hisp/dhis/de/i18n_module_<lang>.properties
+source_file  = dhis-2/dhis-web-dataentry/src/main/resources/org/hisp/dhis/de/i18n_module.properties
```

Comment:
The `dhis-web-dataentry` module was completely removed from the code base, maybe we should remove this entirely?

Reply:
Sounds reasonable, yeah. Since it's Transifex related, @Philip-Larsen-Donnelly do you have any thoughts on this?

Reply:
Agreed. I've pushed a change to remove the whole dataentry section from the transifex config.